packages/state-manager/src/sagas/socket/startConnection/startConnection.saga.ts

```diff
@@ -47,6 +45,8 @@ const log = logger('socket')
 export function subscribe(socket: Socket) {
   return eventChannel<
     | ReturnType<typeof messagesActions.incomingMessages>
+    | ReturnType<typeof messagesActions.responseSendMessagesIds>
```

Comment:
I really don't understand why this was failing only for me.
Action types I added here were giving an error becuase they were not declared